src/lib/state/DY.Lib.State.PKI.fst

```diff
@@ -60,10 +60,10 @@ val is_public_key_for:
 let is_public_key_for #cinvs tr pk pk_type who =
   match pk_type with
   | PkEnc usg -> (
-    is_encryption_key usg (principal_label who) tr pk
+    is_encryption_key (PkdecKey usg empty) (principal_label who) tr pk
```

Comment:
why is the data set to `emtpy` here?
and related: should the `public_key_type` also take the new usage type (tag and data)? (`usage` there is maybe a confusing name?)

Reply:
This is because `data` is mostly useful when keys are derived from a key schedule (e.g. with kdf_expand), I don't think this applies to long-term keys (the one held by PKI and PrivateKeys states) so I left `data = empty`

Reply:
If it doesn't apply to these keys, why do we have that argument at all?
I guess it's for consistency, but if we only ever use `empty` and have to write that argument everywhere, I'm not so convinced.
Also for consistency, I think we should either have the argument for _all_ keys (i.e., also these long-term keys), or just for the ones where we need it (derived keys).
What do you think?

Reply:
I am saying it doesn't apply to long-term keys, not that it doesn't apply for signature keys / public-key encryption keys. We can have ephemeral public-key encryption keys, or public-key encryption keys that are derived from a key schedule, e.g. this is what happens inside the TreeKEM sub-protocol of MLS.

If you prefer we can also add `data` in PKI's `public_key_type` and PrivateKeys' `private_key_type`, although I doubt it will be useful, we never know what might happen in the future!

Reply:
I think, I'd like to have it also for the PKI keys for consistency. 
But this is not blocking, so feel free to leave it as is.

Reply:
Ok!